src/preview/layout.hbs

```diff
@@ -0,0 +1,122 @@
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    {{#if subject}}
+      <title>{{subject}}</title>
+    {{/if}}
+
+    <style type="text/css">
```

Comment:
Can you remove this empty area, it make the page scroll
<img width="1469" alt="image" src="https://user-images.githubusercontent.com/16245250/223302844-a8027066-359e-4ed0-bf6d-ffc3c34fd572.png">